news/2024/2024-11-28-makeship-x-osu-plushies-second-run.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "Makeship x osu!: Limited Edition Plushies 2.0"
+date: 2024-11-28 22:15:00 +0000
+---
+
+Your favourite osu! plushies just came back for one last dance! Come get yours through the crowdfunding campaigns before they're gone for good!
+
+![](/wiki/shared/news/2024-11-28-makeship-x-osu-plushies-second-run/banner.jpg)
+
+You asked, we delivered. Thanks to [Makeship](https://www.makeship.com/), we're bringing back the fan-beloved osu! cookie and pippi plushies one more time!
+
+Unfortunately, Taikonator couldn't join us this time around. Rumors say he's been stuck retry-spamming osu!taiko beatmaps to get as much score as humanly possible, and it turns out he forgot to disable [NoFail](/wiki/Gameplay/Game_modifier/No_Fail) all this time. May he rest in peace.
```

Comment:
sounds like taikonator dies or smth

```suggestion
Unfortunately, Taikonator couldn't join us this time around. Rumours say he's been stuck retry-spamming osu!taiko beatmaps to get as much score as humanly possible, and it turns out he forgot to disable [No Fail](/wiki/Gameplay/Game_modifier/No_Fail) all this time.
```

Reply:
its kinda intentional for extra comedic effect